### PR TITLE
[improve][tuya] Add a tip that resolves an edge case 

### PR DESCRIPTION
**Pull request by 130s**
## Proposed change

This PR adds a small info about an edge case that is missing in the document, without which the integration didn't function.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: None
- Link to parent pull request in the Brands repository: None
- This PR fixes or closes issue: None

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - <s>I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.</s>
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @130s, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20707).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/tuya.markdown [hunk 1]
```diff
@@ -118,7 +118,7 @@ Click the created project to enter the `Project Overview` page and get the `Auth
...
 
-    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields. Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
+    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which `Tuya Smart` allows, may be avoided. See [community.home-assistant.io#353128](https://community.home-assistant.io/t/new-tuya-integration-2021-11-unable-to-log-in/353128/43)). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
```
Comment (frenck) [inline source/_integrations/tuya.markdown]: We should not refer to community guides. If this documentation needs to be improved, we should add it directly here and not link people around.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/tuya.markdown (hunk 1 above)
Comment (130s) [reply source/_integrations/tuya.markdown]: Sure, I have no strong opinion here so removed the reference.

File: source/_integrations/tuya.markdown [hunk 2]
```diff
@@ -118,7 +118,7 @@ Click the created project to enter the `Project Overview` page and get the `Auth
   description: >
     - App account not linked with cloud project: On the [Tuya IoT Platform](https://iot.tuya.com/cloud/), you have linked devices by using Tuya Smart or Smart Life app in your cloud project. For more information, see [Link devices by app account](https://developer.tuya.com/en/docs/iot/Platform_Configuration_smarthome?id=Kamcgamwoevrx&_source=7a356dd493196a01bb9021b7680a2a45#title-3-Link%20devices%20by%20app%20account).
 
-    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields. Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
+    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which `Tuya Smart` app allows, may not work so should be avoided for Home Assistant integration). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
```
Comment (frenck) [inline source/_integrations/tuya.markdown]:
```suggestion
    - Incorrect username or password: Enter the correct account and password of the Tuya Smart or Smart Life app in the **Account** and **Password** fields (social login, which the Tuya Smart app allows, may not work, and thus should be avoided for use with the Home Assistant integration). Note that the app account depends on which app (Tuya Smart or Smart Life) you used to link devices on the [Tuya IoT Platform](https://iot.tuya.com/cloud/).
```

**Review by frenck — APPROVED** [review]
Thanks, @130s 👍

**Pull request merged**